node/node.go

```diff
@@ -851,15 +946,23 @@ func (n *Node) runManager(ctx context.Context, securityConfig *ca.SecurityConfig
 	if err != nil {
 		return false, err
 	}
+	// The done channel is used to signal that the manager has exited.
 	done := make(chan struct{})
+	// runErr is an error closed on by goroutine that runs the manager
```

Comment:
Maybe "`runErr` is the error returned when the manager stops running.  It is declared here because running the manager happens in a goroutine"?